docs/python_client.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Python client
+excerpt: Developing using the MemGPT Python client
+category: 6580dab16cade8003f996d17 
+---
+
+The fastest way to integrate MemGPT with your own Python projects is through the `MemGPT` client class:
+```python
+from memgpt import MemGPT
+
+# Create a MemGPT client object (sets up the persistent state)
+client = MemGPT(
+  quickstart="openai",
+  config={
+    "openai_api_key": "YOUR_API_KEY"
+  }
+)
+
+# You can set many more parameters, this is just a basic example
+agent_id = client.create_agent(
+  agent_config={
+    "persona": "sam_pov",
+    "user": "cs_phd",
+  }
+)
+
+# Now that we have an agent_name identifier, we can send it a message!
+# The response will have data from the MemGPT agent
+my_message = "Hi MemGPT! How's it going?"
+response = client.user_message(agent_id=agent_id, message=my_message)
+```
+
+## More in-depth example of using the MemGPT Python client
```

Comment:
We might want to break this up into several chunks, right now the way it gets formatted in readme (https://memgpt.readme.io/docs/python_client) is kind of annoying since it turns into a scrollview (bad UX)